load/load.py

```diff
@@ -0,0 +1,75 @@
+import os
+import zipfile
+import duckdb
+import tempfile
+from datetime import datetime
+
+# Step 1: Extract the zip file into a temporary directory
+def extract_zip_file(zip_file_path):
+    temp_dir = tempfile.TemporaryDirectory()  # Create a temporary directory
+    with zipfile.ZipFile(zip_file_path, 'r') as zip_ref:
+        zip_ref.extractall(temp_dir.name)
+    return temp_dir
+
+# Step 2: Load JSON files into DuckDB with 'loaded_at' and process only new files
+def load_json_gz_files_in_duck_db(json_dir, duckdb_path):
+    # Open the DuckDB connection (ensure you connect to the correct persistent database)
```

Comment:
Style: I'd suggest avoiding comments unless where necessary. In this case, the code below is self-explanatory. Adding all these comments can make the code less readable than just letting the code speak for itself. This apply to other comments as well that we could remove!

Reply:
I also thought this beforehand but just leave them for more clarity.